.github/workflows/ci.yml

```diff
@@ -56,14 +66,18 @@ jobs:
     name: Eager mode tests for Python ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5.0.0
-        name: Set up Python ${{ matrix.python-version }}
-
+      - uses: mamba-org/setup-micromamba@v1
+        name: Set up Micromamba environment
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+          environment-name: test-env
+          create-args: >-
+            python=${{ matrix.python-version }}
+            pip
```

Comment:
We can use 'uv', familiar with it? Just install uv and use it as 'uv pip install'
Is significantly faster, quite new

Reply:
Yeah, I'm using it locally. It's doable but @matthewfeickert, wasn't there an issue with tensorflow installation? I remember you brought something up at some point on slack

Reply:
@jonas-eschle https://github.com/astral-sh/uv/issues/2062, https://github.com/scikit-hep/pyhf/pull/2444

Reply:
Ok `uv` needs to install in editable mode to install from git. That's stupid

Reply:
`uv` is great, and I'm working to get the `pyhf` CI to use it, but I agree with @ikrommyd that is a whole PR unto itself and atomicly different from this one.

I think you can get ahead of `pyhf` when it comes to using `uv` in CI, but I'll keep you updated as well. :)